### DISCUSSION TITLE
CKB 0.111.0 Full Node (ckb run --indexer)

### DISCUSSION THREAD

**Discussion by joshyates1980** [nervosnetwork/neuron#2848] [General]
It seems **_Neuron-v0.110.2_** will need to be updated before ckb run --indexer will sync with **_ckb_v0.111.0_**. Is this correct?

![image](https://github.com/nervosnetwork/neuron/assets/7255545/9e079dcd-c256-41fb-8399-46e079d2fe82)

Reference: https://github.com/nervosnetwork/ckb/discussions/4157#discussioncomment-7064024

**Comment by yanguoyu** [comment]
Hi @joshyates1980, This is just a reminder, and it does not affect functions. If you run ckb manually, Neuron will check the external ckb version and compare it with Neuron's internal ckb node version, if the major version or minor version is not the same, it will remind users. We will make the reminder more clear by https://github.com/nervosnetwork/neuron/issues/2780.
